home.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/00fad0ddbb.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <div class="social-media">
+        <div></div>
+        <div class="social-media-links">
+          <i class="fa-brands logo-icon fa-square-facebook"></i>
+          <i class="fa-brands logo-icon fa-twitter"></i>
+          <a href="">English</a>
+          <a href="">My Page</a>
+          <a href="">Logout</a>
+        </div>
+      </div>
+
+      <nav class="navbar nav-mobile disappearing-nav1">
+        <div class="hamburger">
+          <div id="myHamburger" class="ham-container hamburger">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+          </div>
+          <div id="myDropdown" class="dropdown-content">
+            <a href="./home.html" class="nav-link">Home</a>
+            <a href="./about.html" class="nav-link">About</a>
+            <a href="#Contact" class="nav-link">Venue</a>
+          </div>
+        </div>
+      </nav>
+
+      <nav class="menu-bar">
+        <div class="topnav">
+          <div class="bx1">
+            <a href="home.html" class="logo-txt"
+              >F<i class="fa-brands fa-google"></i> MusicFest</a
+            >
+          </div>
+          <div class="bx2">
+            <a href="">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Venue</a>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="first-sec">
```

Comment:
- [ ] Kindly ensure that your design matches the Behance design, this includes the background images and the font color used:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/181729687-58f89c77-0697-4831-a5c4-0d49355c4165.png) |  ![image](https://user-images.githubusercontent.com/61408860/181729734-8e49986d-87f4-4846-a773-6d83a2f29e8a.png)